main.js

```diff
@@ -13,4 +18,17 @@ document.body.appendChild($image)
 
 var car1 = new Car($image, 1, 'north', [0, 0])
 
-console.log(car1)
+document.addEventListener('keydown', function (e) {
+  if (e.keyCode == '38') {
```

Comment:
Avoid using `==`. If it's a `Number`, then compare it to a `Number`. It's always better to compare things strictly `===`.